### DISCUSSION TITLE
Обновление NCANode без интернета

### DISCUSSION THREAD

**Discussion by obrolisa** [malikzh/NCANode#188] [Q&A]
У нас возник вопрос от клиента, у которого система установлена в локальной сети без доступа к интернету. Он интересуется, есть ли другие источники, кроме сайта https://ncanode.kz/, откуда можно скачать новую версию NCANode, и как установить обновление без интернет-соединения?

Так как после скачивания файла с вашего сайта, клиенту нужно вызвать специальную команду для установки, которая требует доступа в интернет для загрузки дополнительных файлов. Можете ли вы предоставить информацию о том, где можно получить весь пакет обновления или как иначе выполнить установку без необходимости в интернет-соединении?

Спасибо за ваше внимание.

**Comment by malikzh** [comment]
Здравствуйте. Приложение загружает crl с оффициального сайта нуц. Ссылку
можете увидеть в настройках

С уважением, Malik Zharykov.
*Software Engineer*
E-mail: ***@***.***
GitHub: https://github.com/malikzh
Twitter: https://twitter.com/malikz_h
LinkedIn: https://www.linkedin.com/in/malik-zharykov-bba687165/


Ср, 7 февр. 2024 г. в 19:01, obrolisa ***@***.***>:

**Reply by obrolisa** [reply]
Как вручную установить CRL если NCANode на установленном сервере не имеет доступа к интернету и не может скачать его с официального сайта НУЦ?

**Reply by TrunovK** [reply]
Как вариант при сборке докер контейнера скачать эти фалы, поместить в определенный каталог, в настройках прописать файловые урл до этих файлов. И так получится что при обращении к файлам на скачивание он будет брать их с диска а не с внешнего ресурса. Главное не забывать периодически обновлять эти файлы.
